site/src/modules/notifications/utils.tsx

```diff
@@ -0,0 +1,29 @@
+import EmailIcon from "@mui/icons-material/EmailOutlined";
+import WebhookIcon from "@mui/icons-material/WebhookOutlined";
+
+// TODO: This should be provided by the auto generated types from codersdk
```

Comment:
Can you add this to https://github.com/coder/team-coconut/issues/36 please?
Or are the values available but you haven't wired them up yet?

Reply:
I will add it to https://github.com/coder/team-coconut/issues/36 👍